docs/src/main/asciidoc/rest-client.adoc

```diff
@@ -606,13 +608,14 @@ However, you can change this default behavior and constrain a provider to:
 * only consider *client* calls by adding the `@ConstrainedTo(RuntimeType.CLIENT)` annotation to your provider;
 * only consider *server* calls by adding the `@ConstrainedTo(RuntimeType.SERVER)` annotation to your provider.
 
+[#using-a-mock-http-server-for-tests]
 == Using a Mock HTTP Server for tests
 
 Setting up a mock HTTP server, against which tests are run, is a common testing pattern.
 Examples of such servers are link:http://wiremock.org/[Wiremock] and link:https://docs.hoverfly.io/projects/hoverfly-java/en/latest/index.html[Hoverfly].
 In this section we'll demonstrate how Wiremock can be leveraged for testing the `CountriesService` which was developed above.
 
-First of all, Wiremock needs to be added as a test dependency. For a Maven project that would happen like so:
```

Comment:
Super minor nitpick, but it seems like something in workflow removed spaces in this file

Reply:
hmm... I added newlines in long lines, it seems our autoformatter prefers long lines... and removes newlines instead of replacing them with spaces (which is what the asciidoc "compiler" does) :( will roll that back

Reply:
done